### PR TITLE
Added revproxy deployment docs

### PR DESCRIPTION
**Pull request by timwright12**
## Description
Added documentation on how to deploy the reverse proxy.

Intentionally left out build & deploy instructions as it seems like that is very tied to the operations team.

Closes https://github.com/department-of-veterans-affairs/va.gov-team/issues/7646

### CODE REVIEW

**Review by jhonnyoddball — APPROVED** [review]

File: packages/documentation/src/pages/platform/architecture/reverse-proxy.mdx [hunk 1]
```diff
@@ -37,5 +37,18 @@ Nginx conf files containing the properties for each environment:
...
+
+If you update the revproxy in the devops repository from inside the [/ansible/deployments](https://github.com/department-of-veterans-affairs/devops/tree/master/ansible/deployment) 
+directory you will need to trigger a deployment of the reverse proxy to see your changes in the appropriate environment.
+
+1. Visit the deployment job for the relevant environment: [dev](http://jenkins.vfs.va.gov/job/deploys/job/revproxy-vagov-dev/), [staging](http://jenkins.vfs.va.gov/job/deploys/job/revproxy-vagov-staging/), or [production](http://jenkins.vfs.va.gov/job/deploys/job/revproxy-vagov-prod/)
+2. Find the latest successful deployment; it will be at the top of the dashboard with all steps successful.
+3. Click on the deployment number to visit the landing page for that deployment.
+4. Click "Replay" in the sidebar to deploy your changes. If you don't see, "Replay," "Rebuild" can also be 
+used. If you don't see either option and you think you should please 
+[file an issue with the Operations team](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=operations%2C+devops%2C+needs-grooming&template=ops_issue_template.md&title=).
```
Comment (jbritt1) [inline packages/documentation/src/pages/platform/architecture/reverse-proxy.mdx:52]: Who is the target audience? We don't really give this permission out to the masses so if it is just intended to be for platform engineers it should be fine, but not just anyone who wants these permissions will necessarily get them to my knowledge.

Comment (timwright12) [reply packages/documentation/src/pages/platform/architecture/reverse-proxy.mdx:52]: Very likely just front-end tools and anyone you all feel should have access

Comment (jbritt1) [reply packages/documentation/src/pages/platform/architecture/reverse-proxy.mdx:52]: Roger that, LGTM!

**Review by jbritt1 — APPROVED** [review]
:shipit:

**Pull request merged**